src/Engine/ProtoCore/DSASM/Executable.cs

```diff
@@ -50,6 +50,9 @@ public enum OffsetConstants
         public ProtoCore.DSASM.ProcedureTable[] procedureTable { get; set; }
         public ProtoCore.DSASM.SymbolTable[] runtimeSymbols { get; set; }
 
+        public List<CodeBlock> CodeBlocks { get; set; }
```

Comment:
If we already have `CompleteCodeBlock`, do we still need `CodeBlocks`? They are a little bit confusing, although I know the difference :-)